source/_integrations/growatt_server.markdown

```diff
@@ -63,27 +70,46 @@ To obtain an API token for your Growatt account:
 3. Generate or retrieve your API token.
 4. Use this token during the integration setup in Home Assistant.
 
-Using an API token is recommended for MIN/TLX inverters as it uses the official Growatt API, which offers better stability, support and feature growth.
+If your inverter supports API token, this authentication method is recommended as it uses the official Growatt API, which offers better stability, support and feature growth.
 
-## Prerequisites
+### Compatibility
 
-- Growatt account
-- Login credentials to that Growatt account, you will need them during setup of the integration
+#### Classic API
 
-{% include integrations/config_flow.md %}
+When using username and password authentication the Growatt integration uses the same API as the ShinePhone app. Hence, if your inverter can be controlled via the ShinePhone app, the Growatt integation can access the same data.
+
+#### API token
+
+Authentication using API token is currently supported for the following inverters. For the integration to support additional models, they must first be supported by the [Growatt Python library](https://github.com/indykoning/PyPi_GrowattServer).
+
+**MIC 600-3300TL-X Series:** 600TL-X, 750TL-X, 800TL-X, 1000TL-X, 1500TL-X, 2000TL-X, 3000TL-X, 3300TL-X
+
+**MIN 2500-6000TL-X Series:** 2500TL-X, 3000TL-X, 3600TL-X, 4200TL-X, 4600TL-X, 5000TL-X, 6000TL-X
+
+**MIN 2500-6000TL-XE Series:** 2500TL-XE, 3000TL-XE, 3600TL-XE, 4200TL-XE, 4600TL-XE, 5000TL-XE, 6000TL-XE
+
+**MIN 2500-6000TL-XH Series:** 2500TL-XH, 3600TL-XH, 4200TL-XH, 4600TL-XH, 5000TL-XH, 6000TL-XH
+
+**MIN 2500-6000TL-XA Series:** 2500TL-XA, 3000TL-XA, 3600TL-XA, 4200TL-XA, 4600TL-XA, 5000TL-XA
+
+**MIN 3000-7600TL-XH US Series:** 3000TL-XH US, 3800TL-XH US, 5000TL-XH US, 6000TL-XH US, 7600TL-XH US, 8200TL-XH US, 9000TL-XH US, 10000TL-XH US, 11400TL-XH US
+
+**MOD 3-10KTL3-XH Series:** 3000TL3-XH, 4000TL3-XH, 5000TL3-XH, 6000TL3-XH, 7000TL3-XH, 8000TL3-XH, 9000TL3-XH, 10KTL3-XH
+
+**MID 11-30KTL3-XH Series:** 11KTL3-XH, 12KTL3-XH, 13KTL3-XH, 15KTL3-XH, 17KTL3-XH, 20KTL3-XH, 25KTL3-XH, 30KTL3-XH
 
 ## Known limitations
 
 ### Rate limiting with username/password authentication
 
 The classic API (username/password authentication) has strict rate limits that can result in your account being locked out for up to 24 hours if exceeded. To avoid this issue:
```

Comment:
```suggestion
The classic API (username/password authentication) has strict rate limits that can result in your account being locked out for up to 24 hours if these limits are exceeded. To avoid this issue, use one of the following options:
```